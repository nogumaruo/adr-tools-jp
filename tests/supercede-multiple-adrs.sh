adr new First Record
adr new Second Record
adr new -s 1 -s 2 Third Record
head -9 doc/adr/001-first-record.md
head -9 doc/adr/002-second-record.md
head -13 doc/adr/003-third-record.md
