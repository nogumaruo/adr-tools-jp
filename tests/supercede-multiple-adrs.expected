adr new First Record
doc/adr/001-first-record.md
adr new Second Record
doc/adr/002-second-record.md
adr new -s 1 -s 2 Third Record
doc/adr/003-third-record.md
head -9 doc/adr/001-first-record.md
# 1. First Record

日付: 1992-01-12
決定者: @xxx, @xxx

## ステータス

廃止 by [3. Third Record](003-third-record.md)

head -9 doc/adr/002-second-record.md
# 2. Second Record

日付: 1992-01-12
決定者: @xxx, @xxx

## ステータス

廃止 by [3. Third Record](003-third-record.md)

head -13 doc/adr/003-third-record.md
# 3. Third Record

日付: 1992-01-12
決定者: @xxx, @xxx

## ステータス

提案中

廃止 [1. First Record](001-first-record.md)

廃止 [2. Second Record](002-second-record.md)

